src/OrchardCore.Modules/OrchardCore.Rewrite/Permissions.cs

```diff
@@ -0,0 +1,25 @@
+using OrchardCore.Security.Permissions;
+
+namespace OrchardCore.Rewrite;
+
+public sealed class Permissions : IPermissionProvider
+{
+    public static readonly Permission ManageRewrites = new Permission("ManageRewrites", "Manage rewrites");
```

Comment:
Please move this into it's own class instead of adding it into the permission provider. Put it in a `RewritePermissions` class, and rename this `Permissions` class to RewritePermissionPriovider. Renaming `Permissions` to `RewritePermissionProvider` is not important, since this is what we do in all other modules, but I think its a better practice.

Reply:
Is there any module with such practices that I could use as a reference?

Reply:
https://github.com/OrchardCMS/OrchardCore/blob/e695859a179354d32cfc26771b5395b6423a66fd/src/OrchardCore/OrchardCore.Admin.Abstractions/AdminPermissions.cs#L5

`RewritePermissionProvider` would look like this https://github.com/OrchardCMS/OrchardCore/blob/e695859a179354d32cfc26771b5395b6423a66fd/src/OrchardCore.Modules/OrchardCore.Admin/Permissions.cs